ci/build.yml

```diff
@@ -526,9 +530,9 @@ jobs:
         if [[ "$(Linux.status)" != "Succeeded"
             || "$(macOS.status)" != "Succeeded"
             || "$(Windows.status)" != "Succeeded"
-            || "$(compatibility_linux.status)" != "Succeeded"
-            || "$(compatibility_macos.status)" != "Succeeded"
-            || "$(compatibility_windows.status)" != "Succeeded"
+            || ("$(is_release)" == "false" && "$(compatibility_linux.status)" != "Succeeded")
```

Comment:
Boolean logic is hard. Please double-check this means what I wanted: do not fail on skipped compat jobs for a release.